draft-ietf-rats-corim.md

```diff
@@ -790,25 +791,31 @@ The supply chain entity that is responsible for providing the the measurements (
 is by default the CoRIM signer. If a different entity is authorized to provide measurement values,
 the `authorized-by` statement can be supplied in the `measurement-map`.
 
+An environment may have multiple measured elements.
+Measured elements are distinguished from each other by measurement keys.
+Measurement keys may be used to disambiguate measurements of the same type originating from different elements.
 
 ~~~ cddl
 {::include cddl/measurement-map.cddl}
 ~~~
 
 The following describes each member of the `measurement-map`:
 
-* `mkey` (index 0): An optional unique identifier of the measured
-  (sub-)environment.  See {{sec-comid-mkey}}.
+* `mkey` (index 0): An optional measurement key.
+ See {{sec-comid-mkey}}.
 
-* `mval` (index 1): The measurements associated with the (sub-)environment.
-  Described in {{sec-comid-mval}}.
+* `mval` (index 1): The measurements associated with the environment.
+ Described in {{sec-comid-mval}}.
 
 * `authorized-by` (index 2): The cryptographic identity of the individual or organization that is
- the designated authority for this measurement. For example, producer of the measurement or a delegated supplier.
+ the designated authority for this measurement. For example, the producer of the measurement or a delegate.
 
 ###### Measurement Keys {#sec-comid-mkey}
 
-The types defined for a measurement identifier are OID, UUID or uint.
+Measurement keys are locally scoped identifiers.
+`Mkey` values may be necessary to disambiguate multiple measurements of the same type.
+
+`Mkey` identifiers can be either OID, UUID or uint.
```

Comment:
`mkey` is an extensible type.  This aspect should be highlighted.

Reply:
Have added a line as suggested by Thomas!